dev-requirements.txt

```diff
@@ -4,6 +4,6 @@ gitpython==3.1.37
 ipdb==0.13.13
 isort==5.12.0
 pip-licenses==4.3.3
-pyright==1.1.329
+pyright==v1.1.335
```

Comment:
Added since GHA was complaining about the version. This likely already has been updated by the time I rebase w/ the parent branch